index.js

```diff
@@ -18,9 +18,18 @@ function guessLineEnding(text) {
 }
 
 function parse(text, opts) {
-  const parseFunction = opts.parser === "flow"
-    ? parser.parseWithFlow
-    : parser.parseWithBabylon;
+  let parseFunction
+
+  switch (opts.parser) {
```

Comment:
please use if / else if ... It's a lot shorter and less error prone than switch statements.